packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -63,12 +65,12 @@ type ConfigSchema< T > = {
  * Default configuration.
  */
 const defaultConfig: Partial< SnapWPEnv & SnapWPConfig > = {
+	corsProxyPrefix: '/proxy',
 	graphqlEndpoint: 'index.php?graphql',
-	uploadsDirectory: '/wp-content/uploads',
 	restUrlPrefix: '/wp-json',
-	corsProxyPrefix: '/proxy',
+	uploadsDirectory: '/wp-content/uploads',
 	// eslint-disable-next-line n/no-process-env -- We're using `NODE_ENV` to derive a default value.
-	useCorsProxy: process.env.NODE_ENV === 'development',
+	hasCorsProxy: true,
```

Comment:
If I understand the code correctly, we dont need a default since it's inherited.

```suggestion
```